deploy/helm/sumologic/templates/metrics/otelcol/statefulset.yaml

```diff
@@ -108,6 +108,9 @@ spec:
         args:
           - --log-level={{ .Values.otelcol.metadata.metrics.logLevel }}
           - --config=/etc/otel/config.yaml
+          {{- if eq .Values.otelcol.metrics.enabled false }}
+          - "--metrics-level=none"
```

Comment:
Do we need the quotes? 2 lines above `--config` doesn't need it?

Reply:
we done, changed